admin_guide/upgrading_citus.rst

```diff
@@ -72,6 +72,11 @@ After installing the new package, run the extension upgrade script:
   psql -c '\dx'
   # you should see the newer Citus version in the list
 
+Step 3. (upgrade from 5.x only) Add Co-Location Metadata
```

Comment:
I added the upgrade note as an optional third step rather than a new upgrade classification section. I thought the paragraph is short so it flows better as a little step than as a wimpy section.